README.rst

```diff
@@ -340,6 +340,38 @@ function:
        pytest.fail("+++ Timeout +++")
 
 
+
+Session Timeout
+===============
+
+The above mentioned timeouts are all per test function. You can also set a
+session timeout in seconds. The following example shows a session timeout
+of 10 minutes (600 seconds)::
```

Comment:
Can you describe this a bit more?  I mean, "session timeout" is probably not something that makes sense to a user not very familiar with py.test internals.  I think this paragraph should describe what this timeout applies to, then explain why this is called a "session timeout".

Reply:
I'll attempt a re-write, but feel free to change the prose if it's still awkward.